### DISCUSSION TITLE
Streammaster LXC

### DISCUSSION THREAD

**Discussion by luttman** [tteck/Proxmox#3483] [Ideas]
https://github.com/SenexCrenshaw/StreamMaster
Is an alternative to threadfin.

**Comment by ghost** [comment]
Upvote! Especially now that Threadfin is broken for large M3U files

**Comment by agryts** [comment]
This would be great to have, the newly merged 0.15 version is also a nice update to the software. Upvoted.

**Comment by Crosus97** [comment]
Upvote, Threadfin is deprecated basicly
